tests/aksperiscopeintegration_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/Azure/aks-periscope/pkg/utils"
+)
+
+func TestEndToEndIntegrationSuccessCase(t *testing.T) {
+	expectation := true
+
+	output, err := utils.RunCommandOnContainer("kubectl", "apply", "-f", "../deployment/aks-periscope.yaml", "--validate=false")
+	actual := successfulRun(t, output, err)
+
+	if actual != expectation {
+		t.Errorf("Expected successful return %v but got %v", expectation, actual)
+	}
```

Comment:
At this point the test knows that the deployment yaml was accepted - I guess the next steps would be to wait for the pod to run and then check for the output logs?

Reply:
:bulb: thank you for  message, yeah, this all will change soon thank @Porges got intruding me to `omega` :)